client/plots/matrix.interactivity.js

```diff
@@ -116,6 +117,7 @@ export function setInteractivity(self) {
 					.text('Disco plot')
```

Comment:
line 113 `if()` should not be nested under singleSampleGenomeQuantification

Reply:
Thanks, fixed for both matrix and scatterplot